specs/latest/2.0/index.html

```diff
@@ -3202,6 +3202,60 @@ <h2 class="no-toc">Table of contents</h2>
         <code>FLOAT_32_UNSIGNED_INT_24_8_REV</code>, generates an <code>INVALID_ENUM</code>.
     </p>
 
+    <h3>VertexAttrib function must match shader attribute type</h3>
+
+    <p>
+        If any of the following situations are true, a draw call generates an <code>INVALID_OPERATION</code> error:
```

Comment:
Perhaps clarify to "calls to drawArrays and drawElements". This should also be linked from higher up in the spec, at the point of definition of those APIs.